velox/core/QueryConfig.h

```diff
@@ -303,6 +303,10 @@ class QueryConfig {
   /// The current spark partition id.
   static constexpr const char* kSparkPartitionId = "spark.partition_id";
 
+  /// The config to throw exception on duplicate map entry.
+  static constexpr const char* kSparkThrowExceptionOnDuplicateMapEntry =
+      "spark.throw_exception_on_duplicate_map_entry";
```

Comment:
It's not that correct to use "entry". Suggested name: `spark.throw_exception_on_duplicate_map_keys`

Not sure whether presto needs this config. If it also needs, we should remove "spark." from the name.

Reply:
Updated config name